website/templates/labels/png1/conf.php

```diff
@@ -1,20 +1,38 @@
 <?php
 
-$imgname = "$kret_szablon/label.jpg";
-$img = imagecreatefromjpeg($imgname); /* Attempt to open */
+require_once '../common.php';
```

Comment:
to have functional label on my side (local test with docker + gkShell), I had to put
```
require_once 'common.php';`
```

Reply:
Sounds like black magic, but whatever, changed =)